validator/lib/validateTileset.js

```diff
@@ -95,31 +97,40 @@ function validateTileHierarchy(root, tilesetDirectory) {
         }
 
         if (defined(content) && defined(content.boundingVolume)) {
-            var contentRegion = content.boundingVolume.region;
-            var contentSphere = content.boundingVolume.sphere;
-            var contentBox = content.boundingVolume.box;
-            var tileRegion = tile.boundingVolume.region;
-            var tileSphere = tile.boundingVolume.sphere;
-            var tileBox = tile.boundingVolume.box;
-
-            if (defined(contentRegion) && defined(tileRegion) && !regionInsideRegion(contentRegion, tileRegion)) {
-                return 'content region [' + contentRegion + '] is not within tile region + [' + tileRegion + ']';
+            var contentBV = content.boundingVolume;
+            var tileBV = tile.boundingVolume;
+            var tileTransform = Matrix4.IDENTITY;
+            if(defined(tile.transform)) {
```

Comment:
Just a small style thing, we usually put spaces after the `if`:

`if (defined(tile.transform)) {`